### ISSUE TITLE
agpy.gaussfitter

### ISSUE THREAD

**Issue by navidroo** [keflavich/image_registration#56]
It is necessary to install the agpy library to use gaussfitter in cross_correlation_shifts. However, it doesn't support Python 3.x.x. Is there a workaround?

**Comment by keflavich** [comment]
yes, we can switch to https://github.com/keflavich/gaussfitter - if you want to submit a PR, I'd be happy to accept the contribution
